### ISSUE TITLE
Get channel IDs from env file. 

### ISSUE THREAD

**Issue by MarioDev666** [Million-Token-Community/million-discord-bot#29]
After pulling the new changes to my own repo, and trying to test it on my own server. The bot crashed again due to hardcoded channel IDs. So now i have to hardcode the IDs of my own server, test things, when i am done, ill have to hardocde the IDs of the main server back.
This may sound trivial but i think it would better and more beginner friendly to have the bot pull the IDs from the .env file since those are setup once on heroku for one's own server and thats it.

**Comment by Leon2ev** [comment]
I talked to Peter we need to create a dev/prod config file. After I'm done with youtube I can start with it.

**Comment by MarioDev666** [comment]
You mean update the code to use current .env config file right? or do you have something else in mind?
If we do it using the .env, it would make the setup independent of the files that are on github. This way, a developer can setup their heroku environment, add the key/value pairs for their own server there manually and they no longer have to think about this or change IDs on every push and pull request.

In summary, its going through the code, searching where the channel IDs are being accessed and converting the code to get them from the .env file. What do you think?

**Comment by Leon2ev** [comment]
But for this thing, we don't need .env because it is for sensitive information. How I think is that we can create config.ts file and have there all the ID's and we should initiate all process.env values. And as with .env file, we can make a config example. Finally, based on the current environment prod/dev config will be imported. 

I can make it work.

**Comment by MarioDev666** [comment]
Good points. It's your call at the end. Do it as you think is best and let us know.
Thanks again for the help with the contribution.
Cheers
